packages/runtime/src/lib/constants.ts

```diff
@@ -2,6 +2,7 @@ import { EncodedAssetParams, EncodedLocalStateSchema, EncodedTransaction } from
 
 export const MIN_UINT64 = 0n;
 export const MAX_UINT64 = 0xFFFFFFFFFFFFFFFFn;
+export const MAX_UINT128 = (2n ** 128n) - 1n;
```

Comment:
Shouldn't we use a fixed(constant) value? Won't this be computed each time when we run `checkOverflowUint128`

Reply:
yes